<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fluid3d</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GLPanel" name="GLWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>731</width>
      <height>731</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
   </widget>
   <widget class="QFrame" name="parameterFrame">
    <property name="geometry">
     <rect>
      <x>749</x>
      <y>-1</y>
      <width>441</width>
      <height>731</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="SimulationOptions">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>441</width>
       <height>301</height>
      </rect>
     </property>
     <property name="title">
      <string>Simulation Options</string>
     </property>
     <widget class="QPushButton" name="Simulate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Simulation</string>
      </property>
     </widget>
     <widget class="QLabel" name="timeStep">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>66</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Time Step</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="timeStepEdit">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>96</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0.001</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearScene">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>94</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Scene</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Diffusion Coff</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="diffusionK">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>143</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0.02</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="viscosityK">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>187</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0.2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>190</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Viscosity Coff</string>
      </property>
     </widget>
     <widget class="QLabel" name="De">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Density Magnitude</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>265</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Velocity Magnitude</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="densityMag">
      <property name="geometry">
       <rect>
        <x>152</x>
        <y>226</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>200</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="velocityMag">
      <property name="geometry">
       <rect>
        <x>152</x>
        <y>260</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>20</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="SimulationVar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>421</width>
       <height>361</height>
      </rect>
     </property>
     <property name="title">
      <string>Simulation Variables</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QCheckBox" name="source1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>95</width>
        <height>26</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Source1</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="source2">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>50</y>
        <width>95</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Source2</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="source3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>95</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Source3</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="source4">
      <property name="geometry">
       <rect>
        <x>305</x>
        <y>52</y>
        <width>95</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Source4</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="velSrc1">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>114</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>VelocitySrc1</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="velSrc2">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>116</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>VelocitySrc2</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="velSrc3">
      <property name="geometry">
       <rect>
        <x>257</x>
        <y>117</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>VelocitySrc3</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Clear Scene</string>
   </property>
  </action>
  <action name="actionReset_Everything">
   <property name="text">
    <string>Reset Everything</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLPanel</class>
   <extends>QWidget</extends>
   <header>glpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
